### PR TITLE
XYZ-123: Create new-file

### PR DESCRIPTION
**Pull request by nbaztec**
afsfas

### CODE REVIEW

File: new-file [hunk 1]
```diff
@@ -0,0 +1 @@
+111
```
Comment (nish-cnx) [inline new-file]: What is this?

Comment (marekstachura) [reply new-file]: @nbaztec @nish-cnx how could you approve that?

**Comment by nish-cnx** [comment]
another comment

File: new-file (hunk 1 above)
Comment (nish-cnx) [inline new-file]: Do better

Comment (nish-cnx) [reply new-file]: again

Comment (nish-cnx) [reply new-file]: once

Comment (nish-cnx) [reply new-file]: twice

Comment (nish-cnx) [reply new-file]: thrice

Comment (nish-cnx) [reply new-file]: four

Comment (nish-cnx) [reply new-file]: what

Comment (nish-cnx) [reply new-file]: aaaa

Comment (nish-cnx) [reply new-file]: ascscs

**Pull request closed without merging**